doc/reference/networking/mqtt.rst

```diff
@@ -161,6 +162,9 @@ Note, that TLS credentials referenced by the ``m_sec_tags`` array must be
 registered in the system first. For more information on how to do that, refer
 to :ref:`secure sockets documentation <secure_sockets_interface>`.
 
+Finally, we can optionally set ``set_native_tls`` to enable native TLS support
+instead of offloading TLS operations to the modem.
```

Comment:
```suggestion
Finally, you can optionally set ``set_native_tls`` to enable native TLS support instead of offloading TLS operations to the modem.
```